astropy/units/quantity.py

```diff
@@ -1332,6 +1332,33 @@ def _decompose(self, allowscaledunits=False, bases=[]):
     # https://numpy.org/doc/stable/reference/arrays.ndarray.html#array-conversion
 
     def item(self, *args):
+        """Copy an element of an array to a scalar Quantity and return it.
+
+        Parameters
+        ----------
+        *args
+            - none: in this case, the method only works for arrays with one
+              element (``a.size == 1``), which element is copied into a
+              Quantity scalar object and returned.
+            - int_type: this argument is interpreted as a flat index into the
+              array, specifying which element to copy and return.
+            - tuple of int_types: functions as does a single int_type
```

Comment:
`tuple of int`. I'd write: "indices for all N dimensions of the array specifying the element to copy."